modules/virt-document-attributes.adoc

```diff
@@ -15,6 +15,7 @@
 :VirtVersion: 4.8
 :KubeVirtVersion: v0.41.0
 :HCOVersion: 4.8.2
+//Are these ready to update?
```

Comment:
@ousleyp, PTAL?

Reply:
I have a story for updating these values, so don't worry about this one.

Reply:
Thanks for checking on these!